modules/oe_content_publication/config/post_updates/00004_update_form_display/core.entity_form_display.node.oe_publication.default.yml

```diff
@@ -2,6 +2,7 @@ langcode: en
 status: true
```

Comment:
This file should be in `config/install` and `config/post_updates/00004_...` both.